localstripe/resources.py

```diff
@@ -3277,3 +3277,66 @@ def __init__(self, card=None, customer=None, **kwargs):
 
         self.type = 'card'
         self.card = card_obj
+
+
+class UsageRecord(StripeObject):
+    object = 'usage_record'
+    _id_prefix = 'mbur_'
+
+    class _UsageRecordSummaryWrapper(dict):
+        def _export(self, expand=None):
+            return self
+
+    def __init__(self, subscription_item_id=None, quantity=None,
+                 timestamp=None):
+        self.quantity = quantity
+        self.subscription_item = subscription_item_id
+        self.timestamp = timestamp
+
+        super().__init__()
```

Comment:
Can you add a small `# All exceptions must be raised before this point.` above this line like in the other ressources `__init__`?